server/controllers/partiesController.js

```diff
@@ -22,6 +22,27 @@ class partyController {
       });
     }
   }
+
+  static deleteParty(req, res) {
+    const id = parseInt(req.params.id, 10);
+    let deleted;
+    Parties.map((party, index) => {
```

Comment:
Your implementation here needs to be changed for efficiency. Instead of mapping, since the list of party is an array, why not do a filter and return a response immediately. I would advice you go and read about array filter

Reply:
Okay, the filter method has been implemented.